UKHomeOffice/nmsw-ui
# NMSW-161 Landing page content

# Ticket

NMSW-161

----

## To test

- Run the app
- Go to localhost:3000
- The landing page should match the prototype link attached to the ticket

----

## Notes
